pkg/kdump/kdump-view.jsx

```diff
@@ -281,7 +282,17 @@ export class KdumpPage extends React.Component {
 
     handleSaveClick() {
         return this.props.onSaveSettings(this.state.dialogSettings)
-                .catch(error => Promise.reject(cockpit.format(_("Unable to save settings: $0"), String(error))));
+                .catch(error => {
+                    if (error.message) {
+                        // don't destroy error.details, stack trace, etc.
+                        error.message = cockpit.format(_("Unable to save settings: $0"), error.message);
+                        if (error.details)
+                            error.details = <CodeBlockCode>{ error.details }</CodeBlockCode>;
```

Comment:
FTR: I first tried to add a `className="kdump-error-details"` here, but that completely destroy's PF's own CSS. This component isn't made for custom CSS classes, that's why I wrote the CSS in terms of its default `.pf-c-code-block__pre` class.